tests/src/FunctionalJavascript/CorporateSiteInfoSettingsFormTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\Tests\oe_corporate_site_info\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\Tests\rdf_entity\Traits\RdfDatabaseConnectionTrait;
+
+/**
+ * Tests the JavaScript functionality of Corporate site information config form.
+ */
+class CorporateSiteInfoSettingsFormTest extends WebDriverTestBase {
```

Comment:
I'm afraid we should turn this into a Behat one, this is really stating the business value of the module.